bin/main.rb

```diff
@@ -0,0 +1,101 @@
+# !/usr/bin/env ruby
+require_relative '../lib/game_board'
+require_relative '../lib/game_logic'
+require_relative '../lib/players'
+
+options = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+options_checker = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+def display_game_status(options_p)
+  puts '      *       *      '
+  puts "   #{options_p[0]}  *   #{options_p[1]}   *   #{options_p[2]}  "
+  puts '      *       *       '
+  puts '* * * * * * * * * * * '
+  puts '      *       *       '
+  puts "   #{options_p[3]}  *   #{options_p[4]}   *   #{options_p[5]}   "
+  puts '      *       *       '
+  puts '* * * * * * * * * * * '
+  puts '      *       *       '
+  puts "   #{options_p[6]}  *    #{options_p[7]}  *   #{options_p[8]}   "
+  puts '      *       *       '
+end
+
+def ask_the_name(player)
+  puts "#{player}, please put your name"
+  player_name = gets.chomp
+  if player_name.length >= 2
+    puts "Welcome #{player_name}"
+  else
+    puts 'Please enter a correct name'
+    puts 'Name should be atleast two characters'
+    ask_the_name(player)
+  end
+  player_name
+end
+
+# def play_the_game(player_name, options)
+#   puts "#{player_name}, Please pick an option"
+#   display_game_status(options)
+#   player_option = gets.chomp
+#   player_option.to_i
+# end
+
+puts 'Welcome to the TIC TAC TOE game'
+puts '-------------------------------'
+puts 'Are you ready to play the game ? Y/N'
+
+user_response = gets.chomp
+
+case user_response.upcase
```

Comment:
Use a loop for the play the game, preferably a while loop although any loop would work just fine

Reply:
Thank you, we will update it